src/sdp/DefaultSDP.ts

```diff
@@ -1,21 +1,23 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import SDP from './SDP';
 import SDPCandidateType from './SDPCandidateType';
 import SDPMediaSection from './SDPMediaSection';
 
 /**
- * Implements [[SDP]]. [[SDP]] also includes a few helper functions for parsing string.
+ * [[DefaultSDP]] includes a few helper functions for parsing sdp string.
  */
-export default class DefaultSDP implements SDP {
+export default class DefaultSDP {
```

Comment:
Since there is only one implementation of SDP now, I dont think we should call it default anymore and just call it SDP :)